sympy/printing/latex.py

```diff
@@ -616,22 +616,31 @@ def _print_Pow(self, expr):
             if expr.base.is_Function:
                 return self._print(expr.base, exp=self._print(expr.exp))
             else:
-                tex = r"%s^{%s}"
+                tex = r"%s^%s"
                 return self._helper_print_standard_power(expr, tex)
 
     def _helper_print_standard_power(self, expr, template):
         exp = self._print(expr.exp)
         # issue #12886: add parentheses around superscripts raised
         # to powers
         base = self.parenthesize(expr.base, PRECEDENCE['Pow'])
-        if '^' in base and expr.base.is_Symbol:
-            base = r"\left(%s\right)" % base
+        if '^' in base :
```

Comment:
Checking for substrings doesn't seem very robust. What does the logic for pprint look like?

Reply:
What i understand is that first the expression is parenthesized if necessary then the side brackets **'|'** are added to the picture of expression. And then variables are filled in those brackets.

Reply:
Actually i manipulated the previous written logic to get the parentheses around necessary expression.